db/migrate/20220927073045_create_enrolls.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEnrolls < ActiveRecord::Migration[6.1]
+  def change
+    create_table :enrolls do |t|
+      t.float :progress
+      t.references :user, null: false, foreign_key: true
+      t.references :course, null: false, foreign_key: true
```

Comment:
Thêm `on_delete: :cascade` vào để khi xóa user hoặc course thì enroll tg ứng của user và course cx bay luôn
```suggestion
      t.references :user, null: false, foreign_key: true, on_delete: :cascade
      t.references :course, null: false, foreign_key: true, on_delete: :cascade
```